enable parallel prefill again

kimishpatel opened the pull request:
Summary:
Accidentally the parallel prefill enablement logic fell through the cracks
before

 {F1823815458}

after

 {F1823818585}

Observe the calls to 4x8 kernels

Reviewed By: swolchok

Differential Revision: D61751873

larryliu0820 reviewed (approved):
Would be great to have a test but let's enable this first

kimishpatel committed:
enable parallel prefill again (#4893)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/4893

Accidentally the parallel prefill enablement logic fell through the cracks
before

 {F1823815458}

after

 {F1823818585}

Observe the calls to 4x8 kernels

Reviewed By: swolchok, larryliu0820

Differential Revision: D61751873
